File: app/src/main/java/ovh/trustme/overdated/pojo/Product.java
package ovh.trustme.overdated.pojo;

import com.google.gson.annotations.SerializedName;

public class Product {
    @SerializedName("date")
    public String date;

    @SerializedName("name")
    public String name;


    @Override
    public String toString() {
        return name;
    }

}

File: app/src/main/java/ovh/trustme/overdated/database/Product.kt
package ovh.trustme.overdated.database

import androidx.room.Entity
import androidx.room.PrimaryKey


@Entity(tableName="products_database")
data class Product (

    @PrimaryKey(autoGenerate = true)
    var id: Int,

    var date: String,

    var name: String,

    var dlc_dluo: String,

    var urlImage: String

)
